server/spec/owner_resource_spec.rb

```diff
@@ -30,6 +30,18 @@
     user1.list_owner_consumers(owner1['key']).length.should == 1
   end
 
+  it 'consumer cannot list other consumers for an owner' do
+    owner = create_owner random_string('test_owner')
+    username = random_string("test-user")
+    consumer_name = random_string("test-consumer")
+    user = user_client(owner, username)
+    consumer = consumer_client(user, consumer_name)
+
+    lambda do
```

Comment:
Can we use ``exepect { }`` here to test exceptions. Something like -
```
expect {
      consumer.list_owner_consumers(owner['key'])
}.to raise_error(RestClient::Forbidden)
```

Reply:
Agreed. Updated the code.